redis.conf

```diff
@@ -1266,6 +1266,11 @@ lazyfree-lazy-user-del no
 
 lazyfree-lazy-user-flush no
 
+# Pause clients on write as long as OOM and pending lazy-free jobs in background.
+# By doing so the server might avoid unnecessary eviction or rejection.
+
+client-pause-write-during-oom yes
+
```

Comment:
i'm not certain we want a config for that (even if enabled by default).
why would anyone ever want to turn it off?

Reply:
Because of a bug? Maybe we can hide it from the user somehow?

Reply:
removed client-pause-write-during-oom